### ISSUE TITLE
Netflix Playback Error

### ISSUE THREAD

**Issue by YourTheFinestIveEverKnown** [CastagnaIT/plugin.video.netflix#284]
#### Your Environment
- Netflix add-on version: v0.15.2
- Operating system version/name: Windows 7 Ultimate
- Device model:  PC

Used Operating system:
* [ ] Android
* [ ] iOS
* [ ] Linux
* [ ] OSX
* [ ] Raspberry-Pi
* [X ] Windows

### Describe the bug
Selecting any file to play in Netflix produces the following error message to appear: "Playback Error: Netflix returned the following error: Email or Password is incorrect. Error details have been written to the Kodi Log file."

https://pastebin.com/zjLdQXMZ

#### Expected behavior
Netflix video expected to play

#### Actual behavior
Playback error message appears

#### Steps to reproduce the behavior

1. Browse Netflix Library and select a video
2. 
3. 

### Debug log

The debug log can be found here: 
https://pastebin.com/zjLdQXMZ

### Additional context or screenshots (if appropriate)
https://imgur.com/a/IVaHohC
#### Installation
* [ ] I'm using other Netflix Repo 
* [ ] I'm using a different source 

#### Other information
Removed Netflix Addon, restarted Kodi, Signed back in, getting same error

**Comment by Thunderbird2086** [comment]
15.8 has been release 4 days before.  Please try.  When you start 15.8, please make sure log out from the current Netflix session and login again.

**Comment by CastagnaIT** [comment]
You can try follow:
-Try to change the netflix password and login again
If again not work:
-Try to delete this files in:
C:\Users\[user name]\AppData\Roaming\Kodi\cdm
file .challenge
file .init
file .response
and the subfolder "widevine"

**Comment by YourTheFinestIveEverKnown** [comment]
Issue has been fixed. Thank You. I think all I needed to do was just logout, but I followed your advice anyway, didn't see the challenge or init. files, but whatever. Maybe this will help someone else.

Steps Taken: (May not have all been necessary)
-Disabled Netflix Addon in Kodi
-Closed Kodi and manually deleted the Netflix addon folder along with any netflix files in this directory and went up a level of directories and found some other Netflix file that was also removed (can't remember exactly where it was).
-Started Kodi, installed ver. 15.8 from zip file
-User appeared to still be logged in, selected 'LOG OUT'
-re-entered credentials and it works fine now.
